### PR TITLE
🍡 Marketplace style fixes v2

### PR DESCRIPTION
**Pull request by WRadoslaw**

### CODE REVIEW

File: packages/atlas/src/components/_video/BackgroundVideoPlayer/BackgroundVideoPlayer.styles.ts
Comment (drillprop) [inline packages/atlas/src/components/_video/BackgroundVideoPlayer/BackgroundVideoPlayer.styles.ts:1]: Did you apply the shadow to the buttons? Point 3. I don't see it in dev Chrome.

**Comment by attemka** [comment]
@WRadoslaw 
image

please check the hovers of the avatars

**Review by drillprop — CHANGES_REQUESTED** [review]
Found another issue. One of the icons has a different size for some reason
image

File: packages/atlas/src/components/_video/BackgroundVideoPlayer/BackgroundVideoPlayer.styles.ts [hunk 1]
```diff
@@ -46,6 +46,7 @@ export const ButtonBox = styled.div`
   z-index: ${zIndex.modals};
   display: flex;
   gap: ${sizes(2)};
+  filter: ${cVar('filterEffectElevation1Layer1')};
```
Comment (drillprop) [inline packages/atlas/src/components/_video/BackgroundVideoPlayer/BackgroundVideoPlayer.styles.ts]: filter needs to be applied on the svgs, not on the wrapper of the buttons

**Review by drillprop — APPROVED** [review]
only this small thing left, besides that looks good to me

File: packages/atlas/src/components/NftCarousel/components/MarketplaceCarouselCard/NftCarouselDetails.tsx [hunk 2]
```diff
@@ -176,37 +178,46 @@ export const NftCarouselDetails = ({
         <CSSTransition in={active} timeout={100} classNames={transitions.names.fade} unmountOnExit>
           <InformationContainer isPaused={isPaused}>
             <DetailsContainer>
-              <AvatarGroup spreadAvatars={!smMatch} avatarStrokeColor="transparent" avatars={avatars} />
+              <AvatarGroup spreadAvatars avatarStrokeColor="transparent" avatars={avatars} />
```
Comment (attemka) [inline packages/atlas/src/components/NftCarousel/components/MarketplaceCarouselCard/NftCarouselDetails.tsx:181]: see screenshot above

**Review by attemka — CHANGES_REQUESTED** [review]

**Comment by attemka** [comment]
@WRadoslaw final notes from @KubaMikolajczyk:
> The minor thing - in the timer on hero slider where auction ends in - the colons (:) between hour and minutes and between minutes and seconds should be in color text/default/default - not strong - look at the designs here: https://www.figma.com/file/zQYsZXJo6jmbqcj1kRRmDG/Marketplace?type=design&node-id=464-152611&t=PwKLXqy2LOhqvyRU-4

**Review by attemka — APPROVED** [review]

**Pull request merged**